tutorials/RecSys22tutorial/03_Customize_Merlin_Models.ipynb

```diff
@@ -36,7 +36,7 @@
    "id": "ebc9c8c6-9fab-435f-958b-923b0f3d6ff4",
```

Comment:
<p>I dont know if it fits in the tutorial and how easy it is to explain - but should we add for one model something like model.summary() or model.layers? or how to access/review built models?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/publications/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='77'/>